src/main/java/frc/robot/commands/ToFIntakeCommand.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.ArmSubsystem;
+import frc.robot.subsystems.IntakeSubsystem;
+import frc.robot.subsystems.ArmSubsystem.ArmState;
+import frc.robot.subsystems.IntakeSubsystem.FlightStates;
+import frc.robot.subsystems.IntakeSubsystem.FlightStates;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ToFIntakeCommand extends CommandBase {
+  /** Creates a new ToFIntakeCommand. */
+  private final IntakeSubsystem intakeSubsystem;
+  private final ArmSubsystem armSubsystem;
+  private final double coneKeepSpeed = 0.35; // change after testing
+  private final double cubeKeepSpeed = 0.25; // change after testing
```

Comment:
pull() and push() should already handle this in the subsystem for us

Reply:
pull and push run at a higher speed than we probably need